docs/src/docs/asciidoc/user/getting-started.adoc

```diff
@@ -103,6 +103,17 @@ include::{sourcedir}/impl/src/test/java/org/ehcache/docs/GettingStarted.java[tag
 <3> You define a resource pool for the off-heap.
 <4> You define a resource pool for the disk.
 
+==== Byte Sized Tiering
+
+[source,java,indent=0]
+----
+include::{sourcedir}/impl/src/test/java/org/ehcache/docs/GettingStarted.java[tag=byteSizedTieredCache]
+----
+
+<1> You can also size a tiered cache based on bytes only.You can limit the amount of java heap used by cache for storing key-value pairs. 
+<2> A SizeOfEngine can also be provided which actually ascertains the size of objects on java heap. It uses sizeof library to do so. The configuration for the same takes two arguments. First one specifies the maximum
```

Comment:
Would rephrase that:
The sizing mechanism can be configured along two axis: The first one specifies the maximum number of objects to visit while walking the object graph, the second defines the maximum size of a single object. If the sizing goes above any of these two limits, the mutative operation on the cache will be ignored.